packages/api/endpoints/executions.js

```diff
@@ -106,6 +107,7 @@ async function del(req, res) {
         index: process.env.ES_INDEX,
         ignore: [404],
       });
+      await publishExecutionSnsMessage({});
```

Comment:
So I think that preserving existing behavior would be to send `undefined` on execution deletion (so no arguments to `publishExecutionSnsMessage`), since `publish-executions` is sending `record.dynamodb.NewImage`, which will be undefined for an event where a record was deleted. 

https://github.com/nasa/cumulus/blob/master/packages/api/lambdas/publish-executions.js#L18

I'm also not sure how much this matters, as to whether we need to preserve the existing behavior, because I'm betting it doesn't matter to Cloud Metrics whether they receive an `undefined` message or not. But I'll defer to @nemreid

Reply:
I'll leave it up to you whether you want to keep this @botanical , but if we do keep it then we shouldn't be sending any arguments to `publishExecutionSnsMessage`